### PR TITLE
DNS allow recursion and allow query environment variable support

### PR DESCRIPTION
**Pull request by atomicbaum1**
I've added ALLOW_QUERY and ALLOW_RECURSION environment variables to expose the "allow-query" and "allow-recursion" options and tried to update the README.md with correct descriptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +74,8 @@ Bind caching DNS server based on Debian slim with support for DNS forwarders, in
 | `RETRY_TIME`       | int    | `180`     | (Time in seconds) See [BIND SOA](http://www.zytrax.com/books/dns/ch8/soa.html) |
 | `EXPIRY_TIME`      | int    | `1209600` | (Time in seconds) See [BIND SOA](http://www.zytrax.com/books/dns/ch8/soa.html) |
 | `MAX_CACHE_TIME`   | int    | `10800`   | (Time in seconds) See [BIND SOA](http://www.zytrax.com/books/dns/ch8/soa.html) |
-
+| `ALLOW_QUERY`      | string |           | Specify a comma separated list of IP addresses with optional CIDR mask to allow queries from a specific IP address or ranges of IP addresses. This allows for control over who is allowed to query the DNS server.<br/>Example: `ALLOW_QUERY=192.168.1.0/24,127.0.0.1` |
```
Comment (cytopia) [inline README.md]: What is the default value, if `ALLOW_QUERY` has not been specified?

File: README.md [hunk 2]
```diff
@@ -71,7 +74,8 @@ Bind caching DNS server based on Debian slim with support for DNS forwarders, in
...
+| `ALLOW_RECURSION`  | string |           | Specify a comma separated list of IP addresses with optional CIDR mask to allow queries from a specific IP address or ranges of IP addresses.  This option allows this DNS server to forward a request to another DNS server when an address cannot be resolved. <br/>Example: `ALLOW_RECURSION=192.168.1.0/24,127.0.0.1` |
```
Comment (cytopia) [inline README.md]: What is the default value, if `ALLOW_RECURSTION` has not been specified?

**Comment by omercnet** [comment]
+1

**Comment by bauerj** [comment]
This would be a great feature. Any reason why this is not being merged?

**Comment by travis-south** [comment]
> This would be a great feature. Any reason why this is not being merged?

Yeah, any reason why this hasn't been merged? 

Thanks.

**Comment by travis-south** [comment]
Thanks @atomicbaum1 and @cytopia!

**Pull request merged**